app/views/account/fragments/yourDetails.scala.html

```diff
@@ -6,16 +6,15 @@
 @import scala.collection.immutable.List.empty
 @import com.gu.salesforce.Contact
 @import model.SubscriptionOps._
+@import org.joda.time.LocalDate
 
-@(subscription: Subscription[SubscriptionPlan.PaperPlan], chosenPaperDays: List[PaperDay] = empty, maybeContact: Option[Contact] = None)(extra: Html = Html(""))
+@(subscriptionName: String, plan: SubscriptionPlan.PaperPlan, maybeStartDate: Option[LocalDate], chosenPaperDays: List[PaperDay] = empty, maybeContact: Option[Contact] = None)(extra: Html = Html(""))
```

Comment:
good - less painful way to get around the Subscription of some type erased plan problem